packages/genesys-spark-tokens/dist-new/json/gse-flare-dark-core.json

```diff
@@ -1,437 +0,0 @@
-{
```

Comment:
The `dist-new` folder should not have been committed and is no longer created locally either